src/Platform.Renderers/src/Xamarin.Forms.Platform.UAP/Extensions/WinUIHelpers.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WThickness = Microsoft.UI.Xaml.Thickness;
+using WCornerRadius = Microsoft.UI.Xaml.CornerRadius;
+using WGridLength = Microsoft.UI.Xaml.GridLength;
+using UwpGridUnitType = Microsoft.UI.Xaml.GridUnitType;
+
+namespace Xamarin.Forms.Platform.UWP
+{
+	public static class WinUIHelpers
+	{
+		public static WThickness CreateThickness(double left, double top, double right, double bottom)
```

Comment:
WinUI got rid of all the constructors for there types and just uses setters so I created these helpers which made CTRL F replace infinitely easier